jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/RequestTest.java

```diff
@@ -372,6 +377,37 @@ public boolean handle(org.eclipse.jetty.server.Request request, Response respons
         assertThat(response.getStatus(), is(HttpStatus.OK_200));
     }
 
+    public static Stream<Arguments> localeTests()
+    {
+        return Stream.of(
```

Comment:
There is no need to call `toString()` on the locales that are added to the lists, that just adds confusion.